notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<ul><li>"Learn how to wrap a machine learning model into a Docker container and train in on Vertex AI" --&gt; "Learn how to wrap a Pytorch model into a Docker container and train in on Vertex AI" </li></ul><ol><li>"Learn how to deploy a trained machine learning model on Vertex AI as a REST API and query it"--&gt; "Learn how to deploy a trained Pytorch model on Vertex AI as a REST API and query it"</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks! That makes sense.</p>